buildAndReleaseTask/package.json

```diff
@@ -12,16 +12,16 @@
   "author": "Pulumi",
   "license": "MIT",
   "dependencies": {
-    "axios": "^0.27.2",
-    "azure-devops-node-api": "^11.2.0",
-    "azure-pipelines-task-lib": "^3.3.1",
-    "azure-pipelines-tool-lib": "^1.3.2",
-    "semver": "^7.5.2",
-    "typed-rest-client": "^1.8.9"
+    "axios": "^1.6.2",
```

Comment:
This task uses `axios` very minimally. Specifically, we make an HTTP GET to fetch the latest version of the Pulumi CLI.